supervision/detection/utils.py

```diff
@@ -1481,3 +1481,79 @@ def box_iou_batch_with_jaccard(
         for d_idx, d in enumerate(boxes_detection):
             ious[d_idx, g_idx] = _jaccard(d, g, is_crowd[g_idx])
     return ious
+
+
+def remove_noisy_segments(
+    mask: np.ndarray, connectivity=8, area_thresh: int = 810, dist_thresh: float = 70.0
+) -> np.ndarray:
```

Comment:
Let's ensure that the type hints are consistent with other functions

```suggestion
def remove_noisy_segments(
    mask: npt.NDArray[np.bool_], connectivity: int = 8, area_thresh: int = 810, dist_thresh: float = 70.0
) -> npt.NDArray[np.bool_]:
```